### ISSUE TITLE
throw error when model name does not exist

### ISSUE THREAD

**Issue by blbradley** [alonronin/mockingoose#35]
Hello!

Orgs in an export name, and Organization is the name of the mongoose model.

```
import mockingoose from 'mockingoose'
import Orgs from '../org'

it('should return no orgs if none exist', () => {
  mockingoose.Orgs.toReturn([])

  return Orgs.find().then(result => {
    expect(result).toHaveLength(0)
  })
})
```

throws this error:
```
 FAIL  src/models/__test__/org.test.js
  ● should return no orgs if none exist

    expect(received)[.not].toHaveLength(length)

    Expected value to have a 'length' property that is a number. Received:
      undefined

       6 | 
       7 |   return Orgs.find().then(result => {
    >  8 |     expect(result).toHaveLength(0)
         |                    ^
       9 |   })
      10 | })
      11 | 

      at src/models/__test__/org.test.js:8:20
```

This is the correct test:
```
import mockingoose from 'mockingoose'
import Orgs from '../org'

it('should return no orgs if none exist', () => {
  mockingoose.Organization.toReturn([])

  return Orgs.find().then(result => {
    expect(result).toHaveLength(0)
  })
})
```

Can we get a better error message?

Thanks!

**Comment by blbradley** [comment]
Something like `Mongoose model Orgs does not exist'.

**Comment by alonronin** [comment]
Go for it and submit a PR 😄

**Comment by alonronin** [comment]
I thought about it and I'm not sure I want to check if the module name exists or not, as I want to enable you to write your mocks even if you didn't create your models first.

@blbradley I'd like to hear what you think.

**Comment by dovrosenberg** [comment]
For what it's worth, I spent half a day trying to figure out why my mock find wasn't returning anything, only to determine I'd done this same thing.  Perhaps a warning would help and still let you write the mocks before creating the models.

**Comment by andreialecu** [comment]
This should be fixed by #51.

It introduces new recommended usage:
`mockingoose(model).toReturn(...)` where `model` is the actual Mongoose model class, so you don't need magic strings any more. The name will be retrieved from the model class itself.

Passing a string is still supported, for cases such as the one mentioned by @alonronin.
